cpp/log/test.cpp

```diff
@@ -61,4 +68,76 @@ void test() {
 	t1.join();
 }
 
+/**
+ * Creates a FileSink to also write log Messages
+ * into /tmp/openage-log-test.
+ * Only Messages with a level equals or higher
+ * than level::err will be printed in that file.
+ */
+void file_log_sink () {
+	const char *filelog_name = "/tmp/openage-log-test";
```

Comment:
Sorry, looks like `mstemp` is _not_ the function you're looking for; you'll want to use `tmpnam` instead.

I guess I got irritated by the man page's warning to never use `tmpnam`, and to use `mkstemp` instead. Which is bullshit in our case.